### ISSUE TITLE
Video Block: Additional CSS classes automatically getting removed.

### ISSUE THREAD

**Issue by kaitohm** [Automattic/wp-calypso#47989]
#### Steps to reproduce

1. On a Premium plan site, Start a new page and add a video block.
2. Select a video from your media or upload it. Publish and refresh the editor.
3. Select the video block and add a class on the advance field.
4. Click the Update button and refresh the editor.
5. Select the video block and go to the advanced field. The added CSS class is gone.

#### What happened instead

The CSS class is not on to the resultant code and after refreshing the editor it also disappears on the advanced field.

#### Work around

Move Video bock inside a Group block and add CSS custom class to the Group block Instead.

#### Screenshot / Video

See this internal post for a video from the user who reported this: pbEMJv-l9-p2

#### Context / Source

3509740-zd-woothemes

#### Other notes

There was a similar issue recently where the HTML Anchor on the block would get removed, too:
https://github.com/Automattic/wp-calypso/issues/47254

**Comment by ramonjd** [comment]
I've tried every which way and cannot reproduce this.

Gutenberg: v9.9.3
CoBlocks: 2.5.3
Editing Toolkit: 2.19

Tested:
* both the video block and video press
* with the iframed editor on WPCOM and wp-admin
* uploading a video and embedding an external (YouTube) video
* adding a class name, multiple classnames, using alphanumeric chars and hyphens 
* adding classes before, after and in between the aspect ratio auto-generated classnames, e.g., `wp-has-aspect-ratio`
* by checking the instructions on pbEMJv-l9-p2

At all times the class names persisted in both the editor and on the preview logged in and logged out.

The last relevant [change to the advanced class field for embeds I can find](https://github.com/WordPress/gutenberg/pull/25295) took place in September, 2020. 

It was deployed on WPCOM with the 9.1 upgrade in mid-October. 

🤷 

I think we can close this one for now unless there is something specific that I've missed?
